js/letters.js

```diff
@@ -13,13 +13,28 @@ var counter = {
   "q" : 0,  "r" : 0,  "s" : 0,  "t" : 0,
   "u" : 0,  "v" : 0,  "w" : 0,  "x" : 0,
   "y" : 0,  "z" : 0
-}
+};
 
 function countLetters(counter, sample_text){
-  // FIX ME
+  for (var letter in counter) {
```

Comment:
Although theCounter is recursive, countLetters is not recursive. Can you refactor this so that there's only one function and that function is recursive?